Objects/moduleobject.c

```
@@ -701,6 +701,27 @@ module_getattro(PyModuleObject *m, PyObject *name)
     return NULL;
 }
 
+static PyObject *
+module_str(PyModuleObject *m)
+{
+    PyObject *name, *str;
+
+    name = PyModule_GetNameObject((PyObject *)m);
+    if (name == NULL) {
+        /* XXX PyUnicode_FromStringAndSize is documented as part of
+           the legacy API   */
+        PyErr_Clear();
```

Don't silence all errors. Only `AttributeError`.